client/src/slices/connection/connectionSlice.ts

```diff
@@ -34,10 +36,19 @@ const connectionSlice = createSlice({
       })
       .addCase(createInvitation.fulfilled, (state, action) => {
         state.isLoading = false
-        state.id = action.payload.connection.id
-        state.state = action.payload.connection.state
+        state.outOfBandId = action.payload.outOfBandRecord.id
         state.invitationUrl = action.payload.invitationUrl
       })
+      .addCase(fetchConnectionByOutOfBandId.pending, (state) => {
+        state.isLoading = true
+      })
+      .addCase(fetchConnectionByOutOfBandId.fulfilled, (state, action) => {
+        state.isLoading = false
+        if (action.payload.length) {
```

Comment:
```suggestion
        if (action.payload.length > 0) {
```

Never liked this check, adding `> 0` makes it a bit more explicit.